portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,21 +38,52 @@
 
 
   @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+   Query query = new Query("Task").addSort("text", SortDirection.DESCENDING);
```

Comment:
Hmm, I'm not sure why they appear out of order since it looks like you're correctly sorting by the text of the task (docs here if you haven't seen them: https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/Query.html#addsort_1)

Do you have a screenshot?